bundles/org.openhab.binding.yamahamusiccast/src/main/java/org/openhab/binding/yamahamusiccast/internal/YamahaMusiccastHandler.java

```diff
@@ -451,6 +504,7 @@ private void createChannels(String zone) {
         createChannel(zone, CHANNEL_MUTE, CHANNEL_TYPE_UID_MUTE, "Switch");
         createChannel(zone, CHANNEL_VOLUME, CHANNEL_TYPE_UID_VOLUME, "Dimmer");
         createChannel(zone, CHANNEL_VOLUMEABS, CHANNEL_TYPE_UID_VOLUMEABS, "Number");
+        createChannel(zone, CHANNEL_VOLUMEDB, CHANNEL_TYPE_UID_VOLUMEDB, "Number");
```

Comment:
This could be `Number:Dimensionless` with `Units.DECIBEL`. See https://www.openhab.org/docs/concepts/units-of-measurement.html.

Reply:
Sorry, I am not sure how to implement the Units.DECIBEL. Therefore I only use Number:Dimensionless.